utils/src/check/clobber-functions.c

```diff
@@ -111,12 +111,29 @@ static struct clobber_function clobber_pb_sb_hdr_crc_invalid = {
 	&do_clobber_pb_sb_hdr_crc_invalid,
 };
 
+/*
+ * Clobber the superblock flag by setting to bogus value;
+ */
+static int do_clobber_pb_sb_bad_flag(char *data)
+{
+	global_super->flags |= 0xfc;
+	return super_commit();
+}
+
+static struct clobber_function clobber_pb_sb_bad_flag = {
+	PB_SB_BAD_FLAG,
+	"Sets an invalid flag in the superblock.\n" \
```

Comment:
There are many fields in the superblock, so this is a little ambiguous in the way it's written. More precise would be:

"Sets an invalid value for the flags field in the superblock."